lib/crypto/crypto_keys_subr.c

```diff
@@ -32,7 +32,7 @@
  */
 
 static int import_BN(BIGNUM **, const uint8_t **, size_t *);
-static int export_BN(BIGNUM *, uint8_t **, size_t *, uint32_t *);
+static int export_BN(const BIGNUM *, uint8_t **, size_t *, uint32_t *);
```

Comment:
Yes, the `const` was added between 0.9.3 and 0.9.4.

The reason I was thinking that being compatible with other libraries would be good is that some (libressl, boringssl) are designed as "drop-in replacements" and users might want to drop in such a replacement themselves.  But on further consideration... well, it's *probably* safe to assume that nobody is going to resurrect a broken API from a 17-year-old version of OpenSSL at this point.  So don't worry about it.